code/PostgreSQLSchemaManager.php

```diff
@@ -961,6 +976,28 @@ protected function constraintExists($constraint, $cache = true)
         return self::$cached_constraints[$constraint];
     }
 
+    /**
+     * Retrieve a list of constraints for the provided table name.
+     * @param string $tableName
+     * @return array
+     */
+    private function getConstraintForTable($tableName)
```

Comment:
This is adapted from the above `constraintExists` method. This is only ever going to be used when running a dev build so didn't see much point in caching the results.